### ISSUE TITLE
Podcast Promo colour contrast and column span design fixes

### ISSUE THREAD

**Issue by RichardPK** [BBC-archive/psammead#4212]
Resolved by https://github.com/bbc/psammead/pull/4214

**Describe the bug**
Following a recent a11y swarm, a few design issues were uncovered with the podcast promo component:

- [x]  Image looks like it only spans one column on group A and B breakpoints. It should be bigger. 
- [x] Font weight on summary and 'Episode' text should be 'Normal' 
- [x] Not sure if there is a drop shadow on the white box. There should be.
- [x] Heading fails colour contrast - the grey is too light on visited state. Check colours of bg card as well as link text. Designs were wrong-  should have been using `C_METAL`. Now updated.
           

Designs to assist: https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/dashboard?q=podcast%20promo

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to 'https://bbc.github.io/psammead/?path=/story/components-podcastpromo--default'

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**
